packages/block-library/src/query-loop/save.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * WordPress dependencies
+ */
+import { InnerBlocks } from '@wordpress/block-editor';
+
+export default function QueryLoopSave() {
+	return <InnerBlocks.Content />;
```

Comment:
Should this be "null"? Isn't this a dynamic block?

Reply:
Yes, but its inner blocks are the template for the loop.